lib/index.js

```diff
@@ -6,9 +6,9 @@ import FS from 'fs';
 import { dirname } from 'path';
 import YAML from 'js-yaml';
 
-const makeEngineString = engineNames => (
+const getEngineArgs = engineNames => (
   engineNames.map(language => (`-e ${language}`))
-).join(' ');
+).join(' ').split(' ').filter(i => i);
```

Comment:
Isn't this just the same as `.filter(i => i)`? Or does the join and split on the same string do something in the conversion?